tasks/styles.js

```diff
@@ -1,20 +1,24 @@
 'use strict';
 
+const getOptions = require('../lib/getOptions');
+const registerTaskWithProductionMode = require('../lib/registerTaskWithProductionMode');
+
 module.exports = (gulp, $, pkg) => {
   const reportError = require('../lib/error.js');
 
   // Copyright notice placed at top of compiled CSS
   const copyrightPlaceholder = '/*! #copyright DO NOT REMOVE# */';
-  const copyrightNotice = ['/*!',
-    ' * ' + pkg.title + ' - ' + pkg.description,
-    ' * @version v' + pkg.version,
-    ' * @link ' + pkg.homepage,
-    ' * @author ' + pkg.author,
+  const copyrightNotice = [
+    '/*!',
+    ` * ${pkg.title} - ${pkg.description}`,
+    ` * @version v${pkg.version}`,
+    ` * @link ${pkg.homepage}`,
+    ` * @author ${pkg.author}`,
     ' */',
-    ''].join('\n');
+    '',
```

Comment:
In my testing the line break is not getting added on the closing line of the copyright notice when running `build:production`. Is it possible to leave the newline here when minifying?

Reply:
That one's a bit tricky. The exclamation point in the comment (`/*!`) tells the minifier to leave the comment as-is, but newlines surrounding the comment will still be removed. I added a 'padding' inside the comment, so the block will stand out even when the surrounding `\n`s are gone.